app/models/hubstats/comment.rb

```diff
@@ -8,7 +8,7 @@ class Comment < ActiveRecord::Base
       .group("hubstats_comments.user_id")
     }
 
-    scope :created_since, lambda {|time| where("created_at > ?", time) }
+    scope :created_in_date_range, lambda {|start_date, end_date| where("hubstats_comments.created_at > ? AND hubstats_comments.created_at < ?", start_date, end_date)}
```

Comment:
What if we used BETWEEN? It'd be shorter, and it's inclusive:
http://stackoverflow.com/questions/749615/does-ms-sql-servers-between-include-the-range-boundaries